src/Profile.php

```diff
@@ -2150,6 +2156,60 @@ public function showFormSetupHelpdesk($openform = true, $closeform = true)
         $this->showLegend();
     }
 
+    /**
+     * Print the Security form for a profile
+     *
+     * @param $openform     boolean  open the form (true by default)
+     * @param $closeform    boolean  close the form (true by default)
+     **/
+    public function showFormSecurity($openform = true, $closeform = true)
+    {
```

Comment:
same twig ?

also, the form seem (maybe there's difference) to be the same between group, profile and entity, can we factorize the display ?